tests/h2integrate/test_iron/test_full_iron_process.py

```diff
@@ -31,7 +31,7 @@ def test_iron_simulation(subtests):
         save_plots=False,
         use_profast=True,
         post_processing=False,
-        run_full_simulation=True,
+        run_full_simulation=False,
```

Comment:
Should this stay False? Maybe it was a debugging step. I think it should be True unless we purposefully save off all the files needed to run it successfully. I think a CI test is failing because of this.